### PR TITLE
Add document delete and wipe actions

### PR DESCRIPTION
**Pull request by tstirrat**
Adds the actions and dialogs, menus seem to always be minimal width.

Screen Shot 2020-03-17 at 5 34 37 PM
Screen Shot 2020-03-17 at 5 34 41 PM
Screen Shot 2020-03-17 at 5 34 47 PM

### CODE REVIEW

File: src/components/Firestore/Document.tsx [hunk 1]
```diff
@@ -69,6 +73,16 @@ export const Root: React.FC = () => {
 export const Document: React.FC<{ reference: firestore.DocumentReference }> = ({
   reference,
 }) => {
+  const handleDeleteDocument = async () => {
+    const shouldDelete = await promptDeleteDocument(reference);
+    shouldDelete && reference.delete();
```
Comment (yuchenshi) [inline src/components/Firestore/Document.tsx]: Please add comment for TODO: Recursive delete

File: src/components/Firestore/index.scss [hunk 2]
```diff
@@ -24,6 +24,7 @@
 .Firestore-panels {
   display: flex;
   flex-direction: row !important;
+  height: 400px;
```
Comment (yuchenshi) [inline src/components/Firestore/index.scss]: How would this look on taller screens? Can we use `vh`?

File: src/components/Firestore/dialogs/deleteDocument.tsx [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+import React from 'react';
+
+import { confirm } from '../../DialogQueue';
+
+export const promptDeleteDocument = (reference: firestore.DocumentReference) =>
+  confirm({
+    title: 'Delete data',
+    body: (
+      <>
+        <div className="TODO: Callout">
```
Comment (yuchenshi) [inline src/components/Firestore/dialogs/deleteDocument.tsx]: Friendly ping to use real callouts -- they've been merged.

File: src/components/Firestore/index.scss (hunk 2 above)
Comment (tjlav5) [reply src/components/Firestore/index.scss]: And smaller screens :)

File: src/components/Firestore/dialogs/deleteDocument.tsx (hunk 3 above)
Comment (tstirrat) [reply src/components/Firestore/dialogs/deleteDocument.tsx]: Done

File: src/components/Firestore/index.scss (hunk 2 above)
Comment (tstirrat) [reply src/components/Firestore/index.scss]: For now I have done 300px min with 55vh height. this seems to fill the screen pretty well. We can tweak in another PR

**Pull request merged**